executor/show.go

```diff
@@ -1274,7 +1274,11 @@ func ConstructResultOfShowCreateTable(ctx sessionctx.Context, tableInfo *model.T
 		err = restoreCtx.WriteWithSpecialComments(tidb.FeatureIDTTL, func() error {
 			restoreCtx.WriteKeyWord("TTL_JOB_INTERVAL")
 			restoreCtx.WritePlain("=")
-			restoreCtx.WriteString(tableInfo.TTLInfo.JobInterval)
+			if len(tableInfo.TTLInfo.JobInterval) == 0 {
+				restoreCtx.WriteString("1h")
```

Comment:
Can we introduce something like `DefaultTTLJobInterval`?

Reply:
Good!

Reply:
I declare this const value near the `TTLInfo` struct in `model` :+1: